sql/core/src/main/scala/org/apache/spark/sql/execution/command/SetCommand.scala

```diff
@@ -80,6 +82,22 @@ case class SetCommand(kv: Option[(String, Option[String])])
       }
       (keyValueOutput, runFunc)
 
+    case Some((DYN_ALLOCATION_MIN_EXECUTORS.key, Some(value))) =>
```

Comment:
This is an administration operation.  I am not sure it's a good idea to control this by just run set command for long running service such as thrift server as users can easily run this command, and misuse this feature.

How about expose this via the Rest API?